modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportG2P.pm

```diff
@@ -220,13 +222,12 @@ sub parse_input_file {
     # get data from the line
     my $symbol  = $content->{"gene symbol"};
     my $allelic = $content->{"allelic requirement"};
-    my $mode    = $content->{"mutation consequence"};
     my $phen    = $content->{"disease name"};
     my $id      = $content->{"disease mim"} if $content->{'disease mim'} =~ /[0-9]/;
     my @accns   = split/\;/,$content->{"phenotypes"};
     my $pubmeds = $content->{"pmids"};
-    my $confidence_category = $content->{"confidence category"};
-    my $mutation_consequence = $content->{"mutation consequence"};
+    my $confidence_category = $content->{"confidence"};
+    my $mutation_consequence = $content->{"molecular mechanism"};
```

Comment:
The molecular mechanism is being saved in phenotype_feature_attrib has attrib_type `mutation_consequence`.
G2P doesn't have mutation consequence anymore.

Reply:
Yeah I know, do we want to change the attrib name, I can add the patch here but I am not sure if we want to do that for 115.

Reply:
attribs are chared between dbs/teams
We should add a new attrib_type for the mechanism, probably better to do it for 116
For now we shouldn't be saving mechanism in mutation consequence.

Reply:
A [ticket](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-6784) has been created to implement this.